.travis.yml

```diff
@@ -1,12 +1,16 @@
-notifications:
-  email: false
+---
 language: ruby
+
 script:
-- bundle exec rubocop
-- tests/test.sh
+  - bundle exec rubocop
+  - tests/test.sh
+
 sudo: false
+
 rvm:
-- 2.2
+  - 2.1
+  - 2.3
```

Comment:
I think you need `2.3.1` or similar, they stopped doing just `x.y` and required `.z` at some point.